web/web-dev-server.config.mjs

```diff
@@ -18,8 +23,10 @@ export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
   appIndex: 'index.html',
 
   plugins: [
-    /** Use Hot Module Replacement by uncommenting. Requires @open-wc/dev-server-hmr plugin */
-    // hmr && hmrPlugin({ exclude: ['**/*/node_modules/**/*'], presets: [presets.litElement] }),
+    replace({
+      'process.env.API_HOST': '"testgrid-data.k8s.io"',
```

Comment:
One nit: consider adding something in the README about local development that points to this file. That's how I'd run the UI against a local copy of the API, right?

Reply:
In the future, yes. README update sounds great!

Reply:
I will add it as a TODO